libs/core/src/components/sage-switch/sage-switch.tsx

```diff
@@ -0,0 +1,150 @@
+import { Component, Element, Event, EventEmitter, Host, h, Prop } from '@stencil/core';
+
+@Component({
+  tag: 'sage-switch',
+  styleUrl: 'sage-switch.scss',
+  shadow: true,
+})
+export class SageSwitch {
+  @Element() el: HTMLSageSwitchElement;
+
+  /**
+   * A unique identifier for this input. Associates the input with its label
+   */
+  @Prop() componentId!: string;
+
+  /**
+   * Sets state of input control
+   */
+  @Prop() checked = false;
+
```

Comment:
Not sure if we'll adjust later, but for the checkbox, a similar suggestion came up. We agreed to use "determines..." since "indicates" comes across as more suggestive than a binary true or false.

Full example from checkbox:
> It determines whether or not the checkbox is checked.

Reply:
Yep, that works as well. The phrasing has the same idea behind it.

Reply:
This may be another case where a follow-up is needed once a style guide is in place. 

I like the idea of starting all descriptions using a standard set of verbs, though I'd also suggest we drop any leading nouns, pronouns, types, or conditionals for consistency.

Ex. `Determines`, `Displays`, `Indicates`, `Identifies` vs `It determines`, `String displays`, `When provided`, `A unique`. The prop types should already be rendered via JSDoc, so use in the description seems unnecessary.